scripts/genlocaledata.js

```diff
@@ -0,0 +1,225 @@
+/*
+ * genlocaledata.js - automatically find modules and update code that
+ * uses those modules to have the right require() statements at the
+ * top of them
```

Comment:
This does not seem like an accurate description of the file. Is this correct?

Reply:
oh.. It's wrong. I fixed it.